lessons/pydata/databases/index.ipynb

```diff
@@ -53,16 +53,16 @@
     "\n",
```

Comment:
<p>Tohle je fakt nesympatickej error - to mrví sqlalchemy?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/83/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>